opentelemetry-proto/README.rst

```diff
@@ -7,9 +7,9 @@ OpenTelemetry Python Proto
    :target: https://pypi.org/project/opentelemetry-proto/
 
 This library contains the generated code for OpenTelemetry protobuf data model. The code in the current
-package was generated using the v0.20.0 release_ of opentelemetry-proto.
```

Comment:
Wow is that right, we haven't re-generated in years? Any breaking changes?

Reply:
Almost a [year](https://github.com/open-telemetry/opentelemetry-python/pull/3355) :)